### PR TITLE
Move service list to separate section

### PR DESCRIPTION
**Pull request by victoria-wg2**
The first step of updating the service list.

- Move out service list to a separate section
- Remove 'Default' column
- Add columns for v1 and v2 provisioning API presence

There will be other PRs incoming on expanding the service list.

### CODE REVIEW

File: docs/provision-api-reference.md [hunk 1]
```diff
@@ -114,37 +114,10 @@ removed for this user.
...
-| VOICE_VOICEMAIL                | Voicemail service                                                | N/A                                                   | Y
-| VOICE_VOLTE                    | Voice calls over LTE                                             | N/A                                                   |
-| VOICE_VOWIFI                   | Voice calls over WiFi                                            | N/A                                                   |
-| VOICEMAIL_DISABLE_NOTIFICATION | Disable voicemail notifications                                  | N/A                                                   |
-
 **Error cases:**
 * msisdn unknown
 * service name unknown
+
+### Service list
```
Comment (jorunfa) [inline docs/provision-api-reference.md:121]: Should this be h3 (is it a child of `## UPDATE`? Or h2?
```suggestion
## Service list
```

File: docs/provision-api-reference.md [hunk 2]
```diff
@@ -114,37 +114,10 @@ removed for this user.
...
+
+The list of services available to provision is found here: [list of services](https://docs.wgtwo.com/subscription-profile/subscription-services/)
```
Comment (jorunfa) [inline docs/provision-api-reference.md:123]: Use paths without domain, for when we change the domain name (soon).

```suggestion
The list of services available to provision is found here: [list of services](/subscription-profile/subscription-services/)
```

Comment (jorunfa) [reply docs/provision-api-reference.md:123]: Unsure if you can use relative path, e.g.:
```suggestion
The list of services available to provision is found here: [list of services](./provision-service-list.md)
```

If that works, that would be my preferred solution.

File: docs/provision-api-reference.md (hunk 1 above)
Comment (victoria-wg2) [reply docs/provision-api-reference.md:121]: It actually is a child of update, as that's the only call in the provisioning API v1 that uses this list.

File: docs/provision-api-reference.md (hunk 2 above)
Comment (victoria-wg2) [reply docs/provision-api-reference.md:123]: Your second (preferred) option gave the markdown straight up so it wasn't very pretty, at least locally, but your first suggestion worked fine, so I'll happily grab that! Thank you :pray:

**Review by jorunfa — APPROVED** [review]

**Pull request merged**